### PR TITLE
Triple S of UX in Web3

### PR DESCRIPTION
**Pull request by Maniub102**

### CODE REVIEW

**Review by zlatanpham — COMMENTED** [review]
Looks good. Can you add a short introduction to make the note clearer?

File: Design/UX Research/Triple S of UX in Web3 [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+Tags: ux, blockchain, web3
+Author: Nam Bui
+---
+
+In web3 industry,  we like to keep things simple and sexy, so I will introduce you the Triple S of UX in web3
```
Comment (monotykamary) [inline Design/UX Research/Triple S of UX in Web3:6]: Maybe too simple of an introduction.

**Review by monotykamary — CHANGES_REQUESTED** [review]
The content looks good, and it seems that this topic self-coins the Three S's of UX on Web3. It would be great to see a brief story and introduction as to how you were able to compose these three points from your knowledge and expertise.

File: Design/UX Research/Triple S of UX in Web3 (hunk 1 above)
Comment (Maniub102) [reply Design/UX Research/Triple S of UX in Web3]: Yes, I have just added it. Thank you

**Review by zlatanpham — APPROVED** [review]

**Pull request merged**